lib/models.js

```diff
@@ -144,6 +144,8 @@ class Concept {
     this._system = system;
     this._code = code;
     this._display = display;
+    this._descendingFrom = null;
+    this._includesFrom = null;
```

Comment:
I'm not sure this is the right place to put the _descendingFrom_ and _includesFrom_ markers.  A `Concept` is a concept, and this seems to mix it with a rule.  These notions (_descendingFrom_ and _includesFrom_) are really only valid in the context of a value set definition, while `Concept` is valid in _many_ contexts.